_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
-Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
+Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for confusion.
 
 ### More Translation updates and improvements
 
-Apart from the language improvements mentioned above, some other interesting localization changes happened in this release. [@comradekingu](https://github.com/comradekingu), a frequent contributor to and language checker of NewPipe and other open source projects, has once again taken a close look at our strings. There was, as always, a lot of room for improvement. Our wonderful translators rushed to help and immediately implemented his suggestions.
+On the topic of language, you may find it improved in this release, and in other forms of written NewPipe material. [Allan Nordhøy](https://github.com/comradekingu) landed another pleasant and hopefully indistingishable effort. There was, as always, unmaintained walls leaning on room for improvement. The always wonderful translators have carried those improvements onto all languages and some in turn back into English.
 
-We are also happy to provide **new localizations** in this release: Sardinian, Bengali, Portuguese (Portugal), Neapolitan and Berber.
+Happily there are **new NewPipe languages** in this release: Sardinian, Bengali, Portuguese (Portugal), Neapolitan and Berber.
 
-If you are missing a translation or think a phrase needs adjustment, head over to [Weblate](https://hosted.weblate.org/engage/newpipe/) and help us improve NewPipe's translation even more!
+Improving matters is as easy as heading over to [Weblate](https://hosted.weblate.org/engage/newpipe/) to fix even the source strings.
 
 ## Fixes
 
-If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+Next to the now working offline search input, the actual text of the suggestions look better now. [@TobiGr](https://github.com/TobiGr) changed the colours to the NewPipe palette.
 
-An empty error report in certain kinds of crashes is not empty anymore, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is.
```

Comment:
Empty error report, can anything be done with empty error report. The mundane is the mundane. I feel empty reading this.

Reply:
Doesn't matter. We mention every PR in the blog post no matter how small it is, because we want to show appreciation for every little contribution. This will not change.